index.html

```diff
@@ -6,15 +6,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio Microverse</title>
     <link rel="stylesheet" href="styles/styles.css" />
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <nav>
-        <ul>
-          <li class="navbar-link"><a href="#">Portfolio</a></li>
-          <li class="navbar-link"><a href="#">About</a></li>
-          <li class="navbar-link"><a href="#">Contact</a></li>
-        </ul>
-    </nav>
+
+    <header>
+      <!-- NAVBAR -->
+  
+      <nav>
+          <ul>
+            <li class="navbar-link"><a href="#">Welcome</a></li>
+            <li class="navbar-link"><a href="#"><img src="icons/menu-icon.png" alt="Icon"></a></li>
+          </ul>
+      </nav>
+  
+      <!-- HEADLINE -->
+  
+      <div class="headline">
+        <h1>
+          <span class="dark-blue">Hey there. I'm Javier</span>
+          <span class="green">I'm a software developer</span>
+        </h1>
+        <p>
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>
```

Comment:
- [ ] Your portfolio is looking great! Let's make it even better by using the suggested font from the Figma template:

<img width="740" alt="Screen Shot 2022-08-18 at 3 09 45 PM" src="https://user-images.githubusercontent.com/83717285/185486483-5e145a32-bb99-4a58-b016-4ab4d0d9ba14.png">